app/classes/api/name_api.rb

```diff
@@ -263,7 +263,7 @@ def change_deprecated(name)
 
     def add_synonym(name)
       return unless @synonymize_with
-      raise(CanOnlySynonymizeUnsynonimizedNames.new) if name.synonym_id
+      raise(CanOnlySynonymizeUnsynonymizedNames.new) if name.synonym_id
```

Comment:
GitHub claims that this line is changed. I don't see a difference. Am I missing something?

Reply:
Yes. I missed the spelling correction of Unsynon<ins>**i**</ins>mize to Unsynon<ins>**y**</ins>mize